packages/magmo-wallet-client/src/wallet-types.ts

```diff
@@ -0,0 +1 @@
+export type WalletProcess = 'IndirectFunding' | 'DirectFunding';
```

Comment:
As per our discussion, this probably belongs in the `wallet` package.

Reply:
This has been moved to a file in the wallet and `magmo-wallet-client` passes procedure as as string